doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -30,10 +57,51 @@
+    - At least one worker node per availability zone or fault domain to ensure the high availability and resilience of workloads managed by Kubernetes
+- Master node services, including etcd, and worker node services (e.g. consumer workloads) must be kept separate - i.e. there must be at least one master node, and at least one worker node
+- Workloads must ***not*** rely on the availability of the master nodes for the successful execution of their functionality (i.e. loss of the master nodes may affect non-functional behaviours such as healing and scaling, but components that are already running will continue to do so without issue)
+- The following kubelet features must be enabled
+    - CPU Manager
+    - Device Plugin
+    - Topology Manager
+
+All kubelet features can be enabled/disabled by using the `feature-gates:` section in the kubelet config file.  e.g.
+```
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+feature-gates:
+  CPUManager: true|false (BETA - default=true)
+  DevicePlugins: true|false (BETA - default=true)
+  TopologyManager: true|false (ALPHA - default=false)
+```
+
 <a name="4.4"></a>
 ## 4.4 Container runtimes
 
-> This chapter should describe which container runtimes are part of the Reference Architecture.
+In order to support `req.inf.com.03`, the chosen runtime must be compliant with the [Kubernetes Container Runtime Interface (CRI)](https://kubernetes.io/blog/2016/12/container-runtime-interface-cri-in-kubernetes/) and the [Open Container Initiative (OCI) runtime spec](https://github.com/opencontainers/runtime-spec). Examples of container runtimes that are compliant with these specification are (note this is not a complete list and in no particular order):
+- container-d (with CRI plugin enabled, which it is by default)
+- rkt
+- Docker CE (via the dockershim, which is currently built in to the kubelet)
+- CRI-O
+- Frakti
+
+These specifications cover the [full lifecycle of a container](https://github.com/opencontainers/runtime-spec/blob/master/runtime.md#lifecycle) `creating > created > running > stopped` which includes the use of storage that is required during this lifecycle - this is management of the Host OS filesystem by the container runtime. This lifecycle management by the container runtime (when compliant with the above specifications) supports the requirement `req.inf.stg.06` for ephemeral storage for Pods.
+
+If privileged containers are required (not recommended, but required for some features such as the use of CSI drivers) then a container runtime that provides additional security isolation is required in order to support `req.sec.gen.01` and `req.sec.gen.04`.  This could be via the use of lightweight virtual machines (e.g. Kata) or alternative sandbox methods (e.g. gVisor).  For more information on isolation of workloads and underlying Host OS kernel, see the [Security chapter](./chapter06.md).
```

Comment:
IMO this also falls into the "hard-coding implementation choice into the architecture" category
controlling the availability of such run-time features on the K8s level is perfectly possible, and suitable via K8s PodSecurityPolicies
I would argue it is even the preferred way

Reply:
Good point - I have removed this line for now, as Security chapter will deal with the details.  OK to resolve?